packages/react-form/src/hooks/list/hooks/reducer.ts

```diff
@@ -176,8 +176,19 @@ function reduceList<Item extends object>(
     }
     case 'moveFieldItem': {
       const newList = [...state.list];
-      const [item] = newList.splice(action.payload.oldIndex, 1);
-      newList.splice(action.payload.newIndex, 0, item);
+
+      if (
+        action.payload.fromIndex < state.list.length ||
```

Comment:
This might still pass if the index is greater than the length since it will satisfy `action.payload.fromIndex >= 0`

Reply:
yes, sorry, just noticed that

Reply:
No problem! 🙂